cg/store/models.py

```diff
@@ -1189,3 +1191,15 @@ class PacbioSampleSequencingMetrics(SampleRunMetrics):
     polymerase_mean_read_length: Mapped[BigInt | None]
 
     __mapper_args__ = {"polymorphic_identity": DeviceType.PACBIO}
+
+
+class OrderTypeApplication(Base):
+    """Maps an order type to its allowed applications"""
+
+    __tablename__ = "order_type_application"
+
+    order_type: Mapped[OrderType] = mapped_column(sqlalchemy.Enum(OrderType), primary_key=True)
+    application_id: Mapped[int] = mapped_column(
+        ForeignKey("application.id", ondelete="CASCADE"), primary_key=True
+    )
```

Comment:
will this delete the application when deleted? Or is this deleted when the application is removed from the application table?

Reply:
Deleting the application will delete entries in OrderTypeApplication but not the other way around!